test/e2e/testsuite/testsuite.go

```diff
@@ -238,7 +238,6 @@ func InitializeScenario(s *godog.ScenarioContext) {
 					fmt.Println(err)
 					os.Exit(1)
 				}
-
```

Comment:
@jsliacan I don't think just removing this line is doing what you described in the commit log.

Reply:
Oh great, I must've mingled the commits when rearranging after last changes. Thanks for noticing.